internal/core/common/ecs/ecs.go

```diff
@@ -0,0 +1,40 @@
+package ecs
+
+import "strings"
+
+type TaskMetadata struct {
+	ClusterName	string `json:"Cluster"`
+	TaskARN		string `json:"TaskARN"`
+	Family		string `json:"Family"`
+	Revision	string `json:"Revision"`
+	KnownStatus	string `json:"KnownStatus"`
+	Containers  []Container `json:"Containers"`
+}
+
+func (task *TaskMetadata) GetDimensions() map[string]string {
+	dims := make(map[string]string, 0)
+	if idx := strings.Index(task.ClusterName, "/"); idx >= 0 {
+		dims["cluster"] = task.ClusterName[idx+1: len(task.ClusterName)]
+	} else {
+		dims["cluster"] = task.ClusterName
+	}
+	dims["ecs-task-arn"] = task.TaskARN
+	dims["ecs-task-group"] = task.Family
+	dims["ecs_task_version"] = task.Revision
```

Comment:
Inconsistent between kebab-case and snake_case.

Reply:
Also, the ECS standard label config uses snake_case so this one is right and the other two should change.

We use the titlecased `ClusterName` for the task's cluster name field so that it matches what CloudWatch metrics use.